tests/sentry/api/test_invite_helper.py

```diff
@@ -27,16 +22,17 @@ def setUp(self):
             organization=self.org,
             teams=[self.team],
         )
-        self.auth_provider_inst = AuthProvider(
-            provider="Friendly IdP", organization_id=self.organization.id
-        )
-
         self.request = HttpRequest()
+        # Needed for audit logs
+        self.request.META["REMOTE_ADDR"] = "127.0.0.1"
         self.request.user = self.user
 
-    @patch("sentry.api.invite_helper.create_audit_entry")
-    @patch("sentry.api.invite_helper.RpcOrganizationMember.get_audit_log_metadata")
-    def test_accept_invite(self, get_audit, create_audit):
+    def _get_om_from_accepting_invite(self) -> OrganizationMember:
```

Comment:
From my understanding, the tests all seemed to be the same workflow, just with different models attached to the org to determine if SSO is present, and whether or not it is required. I looped that all into one method here, to hopefully make them easier to understand.